http-api-samples/http-api-owner-sample/src/main/java/org/fido/iot/api/OwnerDeviceTypeCriteriaServlet.java

```diff
@@ -56,8 +56,8 @@ protected void doDelete(HttpServletRequest req, HttpServletResponse resp)
     }
 
     try {
-      DataSource ds = (DataSource) getServletContext().getAttribute("datasource");
-      new OwnerDbManager().removeDeviceTypeCriteria(ds, deviceType);
+      //DataSource ds = (DataSource) getServletContext().getAttribute("datasource");
```

Comment:
comment why these are commented out? If they are really not needed then can we just delete them?